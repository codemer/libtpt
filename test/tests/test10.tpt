@set(x, 100)\
@set(y, 0)\
@set(z, 16)\
@set(result, 64 * (x + y) / -z)\
x = ${x}
y = ${y}
z = ${z}
64 * (x + y) / -z = ${result}

