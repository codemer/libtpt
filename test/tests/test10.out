x = 100
y = 0
z = 16
64 * (x + y) / -z = -400

